spark/v3.3/spark-runtime/src/integration/java/org/apache/iceberg/spark/SmokeTest.java

```diff
@@ -66,25 +64,25 @@ public void testGettingStarted() throws IOException {
     sql(
         "CREATE TABLE updates (id bigint, data string) USING parquet LOCATION '%s'",
```

Comment:
nit: for this and the create table statement above, can we change to `USING iceberg` instead?